src/native/external/zlib-ng/deflate_slow.c

```diff
@@ -78,7 +78,8 @@ Z_INTERNAL block_state deflate_slow(deflate_state *s, int flush) {
             unsigned int max_insert = s->strstart + s->lookahead - STD_MIN_MATCH;
             /* Do not insert strings in hash table beyond this. */
 
-            check_match(s, s->strstart-1, s->prev_match, s->prev_length);
+            Assert((s->strstart-1) <= _UI16_MAX,"strstart-1 should fit in uint16_t");
```

Comment:
Unix platforms don't have `_UI16_MAX`. Maybe I could use `USHORT_MAX`?

Reply:
Trying with UINT16_MAX

Reply:
`USHRT_MAX` should work if you include `<limits.h>`

https://en.cppreference.com/w/c/types/limits

Reply:
Thanks. If UINT16_MAX doesn't work, I'll use USHRT_MAX with limits.h

Reply:
Seems this worked, but it uncovered other similar errors.